info2439/module5/assignment5.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Assignment 5</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    <link rel="stylesheet" href="theme/themes/ctheme1.css" />
+    <link rel="stylesheet" href="theme/themes/jquery.mobile.icons.min.css" />
+</head>
+<body>
+<!-- Owen Larson
+     assignment5.html
+     INFO2439
+     Thoendel
+     10/20/19
+-->
+<div data-role="header">
+    <a data-ajax="false" data-position="fixed" href="http://student.mccinfo.net/~owlarson/info2439/index.html" class="ui-btn-left" target="_blank" data-icon="home">Home</a>
+    <h1>Owen Larson</h1>
+</div> <!--Closing Header Div Tag-->
+<div data-role="main">
+    <h2>Assignment #5, <br>
+    Sage Bistro Static Google Map</h2>
+    <div class="ui-grid-a">
+        <div class="ui-block-a">
+            <p>
+                Welcome to the Sage Bistro! We'd love to see you at our restaurant. <br>
+                To find your way here, head over to Omaha, Nebraska and meet us along Sorenson Parkway!
+            </p>
+        </div> <!--Closing ui-block-a Div-->
+        <div class="ui-block-b">
+            <img alt="map" src="https://maps.googleapis.com/maps/api/staticmap?center=Sage+Student+Bistro,Omaha,NE&amp;zoom=17&amp;size=150x150&amp;maptype=roadmap&amp;markers=color:blue%7Clabel:S%7C40.41.304729,-95.959506&amp;key=AIzaSyAFQid4GkVK5cIqrk3n8kuRkAo7YcNeTzo">
```

Comment:
This looks like you used my API key. I would then expect some screenshots either here in your repo or in your BB submission showing that you created your own credentials without billing attached.